<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUpload2</class>
 <widget class="QDialog" name="NewUpload2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1951</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 105, 159);</string>
  </property>
  <widget class="QLabel" name="Orignial_viewer">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>110</y>
     <width>930</width>
     <height>540</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>39</y>
     <width>831</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="fakeImageLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>    Fake Image:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fakeImage_lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BrowseButton_fake">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="UploadButton_fake">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>69</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(144, 170, 255);</string>
   </property>
   <property name="text">
    <string>    Fake Image  </string>
   </property>
  </widget>
  <widget class="QLabel" name="Fake_viewer">
   <property name="geometry">
    <rect>
     <x>952</x>
     <y>110</y>
     <width>930</width>
     <height>540</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>79</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(144, 170, 255);</string>
   </property>
   <property name="text">
    <string>    Original Image  </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>-1</y>
     <width>831</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="originalImageLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Original Image:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="OriginalImage_lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BrowseButton_Original">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="UploadButton_Original">
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 170, 255);</string>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>30</y>
     <width>85</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(144, 170, 255);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>110</y>
     <width>20</width>
     <height>531</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewImageName_lineEdit">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(144, 170, 255);</string>
   </property>
   <property name="text">
    <string>Name of New File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Done_btn">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>30</y>
     <width>85</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(144, 170, 255);</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
